integration_tests/sdk/setup_integration.py

```diff
@@ -242,21 +275,32 @@ def setup_storage_layer(client: Client) -> None:
             else:
                 break
 
+        # TODO: REMOVE, gives the server time to migrate.
+        time.sleep(30)
+
+
```

Comment:
This hack is unacceptable. I need to build https://linear.app/aqueducthq/issue/ENG-2540/add-the-ability-to-wait-for-metadata-store-migration-from-the-sdk before this can merge. cc: @saurav-c